testcafe/tests/drive/public-viewer-feature.js

```diff
@@ -225,18 +225,18 @@ test(`${TEST_PUBLIC_VIEWER_AUDIO}`, async t => {
     width: 100,
     y: 623
   })
-  await publicViewerPage.openFileAndcheckCommonViewerDownload(
+  await publicViewerPage.openFileAndCheckPublicViewer(
     `${FEATURE_PREFIX}/${TEST_PUBLIC_VIEWER_AUDIO}-1`,
     data.FILE_AUDIO,
     'audio',
     true
   )
   //mask on loading bar for mobile
   await t.fixtureCtx.vr.setMaksCoordonnates({
-    height: 25,
-    x: 199,
-    width: 90,
-    y: 409
+    height: 27,
+    x: 160,
+    width: 108,
+    y: 415
   })
```

Comment:
is it wanted in this PR ?

Reply:
I saw my masks were a bit off when checking my screenshots, so I modify the mask.
If you prefer, I can remove the change here and add a new PR for fixing the masks

Reply:
Yes please. 

One PR on goal

Reply:
Done : https://github.com/cozy/cozy-drive/pull/1538